src/OceanSeaIceModels/time_step_ocean_sea_ice_model.jl

```diff
@@ -28,28 +28,26 @@ function time_step!(coupled_model::OceanSeaIceModel, Δt; callbacks=[], compute_
         time_step!(sea_ice)
     end
 
-    ocean.Δt = Δt
-
     # TODO after ice time-step:
     #   - Adjust ocean heat flux if the ice completely melts?
 
+    ocean.Δt = Δt
     time_step!(ocean)
 
     # TODO:
     # - Store fractional ice-free / ice-covered _time_ for more
     #   accurate flux computation?
     tick!(coupled_model.clock, Δt)
-    
     update_state!(coupled_model, callbacks; compute_tendencies)
     
     return nothing
 end
 
-function update_state!(coupled_model::OceanSeaIceModel, callbacks=[]; compute_tendencies=false)
+function update_state!(coupled_model::OceanSeaIceModel, callbacks=[]; compute_tendencies=true)
```

Comment:
do we need compute_tendencies here?

Reply:
I'll test that, we need if it is used in `Simulations` in `Oceananigans`